docker/build.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+export HERMES_DISTRIBUTION=1
```

Comment:
change it to gradle argument, why play iwht sys variables when flags are available in build system?